core/src/main/clojure/xtdb/operator/group_by.clj

```diff
@@ -564,10 +564,9 @@
                                            :args [{:op :local, :local acc-local}
                                                   {:op :local, :local val-local}]}})))
 
-(defmethod ->aggregate-factory :all [agg-opts] (bool-agg-factory :and agg-opts))
-(defmethod ->aggregate-factory :every [agg-opts] (->aggregate-factory (assoc agg-opts :f :all)))
-(defmethod ->aggregate-factory :any [agg-opts] (bool-agg-factory :or agg-opts))
-(defmethod ->aggregate-factory :some [agg-opts] (->aggregate-factory (assoc agg-opts :f :any)))
+(defmethod ->aggregate-factory :bool_and [agg-opts] (bool-agg-factory :and agg-opts))
+(defmethod ->aggregate-factory :every [agg-opts] (->aggregate-factory (assoc agg-opts :f :bool_and)))
```

Comment:
let's drop `EVERY` here too, seems inconsistent to leave that on its own just because the SQL spec doesn't include it in `ANY`/`ALL`/`SOME`

Reply:
I thought about it, but I left it because PostgreSQL does have it: https://www.postgresql.org/docs/9.5/functions-aggregate.html